pom.xml

```diff
@@ -58,12 +58,13 @@ under the License.
 	</modules>
 
 	<properties>
-		<flink.version>1.16.1</flink.version>
-		<flink.shaded.version>15.0</flink.shaded.version>
+		<flink.version>1.17.0</flink.version>
+		<flink.shaded.version>16.1</flink.shaded.version>
 		
 		<jackson-bom.version>2.13.4.20221013</jackson-bom.version>
-		<junit5.version>5.8.1</junit5.version>
+		<junit5.version>5.9.2</junit5.version>
 		<assertj.version>3.21.0</assertj.version>
+		<archunit.version>1.0.1</archunit.version>
```

Comment:
You can actually completely remove this, it's not needed because there's no explicit usage of the archunit version in the repo anymore

Reply:
thanks, makes sense